atlp-rwanda/atlp-devpulse-bn
# remove package-lock from the repo"

### **What does this PR do?**
this PR removes package-lock from repo
adding package-lock.json in gitignore

## How should this be manually tested?

- `Clone the repo`

- `checkout` to new branch 
- - Install all packages by running `npm install`
- push your new branch
- you should find that package-lock is ignored while pushing to remote repo
